gcc/rust/ast/rust-ast.h

```diff
@@ -861,7 +861,9 @@ class Item : public Stmt
   add_crate_name (std::vector<std::string> &names ATTRIBUTE_UNUSED) const
   {}
 
-  bool is_item () const override final { return true; }
+  // FIXME: ARTHUR: Is it okay to have removed that final? Is it *required*
+  // behavior that we have items that can also be expressions?
+  bool is_item () const override { return true; }
```

Comment:
It might be worth putting the final back in to see if it still works as this will enforce that is_item is not overridden anymore though I am not too worried about it.

Reply:
The test suite doesn't pass properly without it sadly. The checks in the various visitors are not enough, so I think it's necessary to keep it. I can change the comment for one meaning we *have* to look at fixing this at some point